src/components/CommentsList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../utils";
+import { Comments } from "./Comments";
+
+const CommentsList = () =>{
+    const { article_id } = useParams();
+    const [currentComments, setCurrentComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+    useEffect(() => {
+        getComments(article_id).then((res) => {
+          setCurrentComments(res);
+          setIsLoading(false);
+        });
+      }, []);
+      if (isLoading) {
+        return <p className="loading">Loading...</p>;
+      } else
+    return (
+        <div>
+            <Comments comments={currentComments}/>
```

Comment:
Good to create a separate component for the comments; stops the components getting too big and bloated. I'd consider mapping here in Comments List and returning the Comment component from the map. That way you would only need to render it if there were comments. You could consider some conditional logic here that checks if the array is empty and returns something different in the UI to say that there are no comments.